modules/gdscript/gdscript_parser.cpp

```diff
@@ -2774,6 +2774,25 @@ GDScriptParser::ExpressionNode *GDScriptParser::parse_get_node(ExpressionNode *p
 			get_node->chain.push_back(identifier);
 		} while (match(GDScriptTokenizer::Token::SLASH));
 		return get_node;
+	} else if (current.type == GDScriptTokenizer::Token::SLASH) {
+		advance();
```

Comment:
It would be more semantic to use `match()` here:

```suggestion
	} else if (match(GDScriptTokenizer::Token::SLASH)) {
```